libs_changes.md

```diff
@@ -0,0 +1,115 @@
+# RFC guidelines - libraries sub-team
+
+# Motivation
+
+* RFCs are heavyweight:
+    * RFCs generally take at minimum 2 weeks from posting to land. In
+      practice it can be more on the order of months for particularly
+      controversial changes.
+    * RFCs are a lot of effort to write; especially for non-native speakers or
+      for members of the community whose strengths are more technical than literary.
+    * RFCs may involve pre-RFCs and several rewrites to accommodate feedback.
+    * RFCs require a dedicated shepherd to herd the community and author towards
+      consensus.
+    * RFCs require review from a majority of the subteam, as well as an official
+      vote.
+    * RFCs can't be downgraded based on their complexity. Full process always applies.
+      Easy RFCs may certainly land faster, though.
+    * RFCs can be very abstract and hard to grok the consequences of (no implementation).
+
+* PRs are low *overhead* but potentially expensive nonetheless:
+    * Easy PRs can get insta-merged by any rust-lang contributor.
+    * Harder PRs can be easily escalated. You can ping subject-matter experts for second
+      opinions. Ping the whole team!
+    * Easier to grok the full consequences. Lots of tests and Crater to save the day.
+    * PRs can be accepted optimistically with bors, buildbot, and the trains to guard
+      us from major mistakes making it into stable. The size of the nightly community
+      at this point in time can still mean major community breakage regardless of trains,
+      however.
+    * HOWEVER: Big PRs can be a lot of work to make only to have that work rejected for
+      details that could have been hashed out first. *This is the motivation for
+      having RFCs*.
```

Comment:
This is one of the motivations for RFCs, but not the only one: actually spelling out and motivating a significant design and its implications is really useful, as is the broad discussion RFCs tend to produce. In short: RFCs often make for better designs.